src/ert/config/_read_summary.py

```diff
@@ -427,7 +441,14 @@ def read_params() -> None:
         if last_params is not None:
             vals = _check_vals("PARAMS", summary, last_params.read_array())
             values.append(vals[indices])
-            dates.append(start_date + unit.make_delta(float(vals[date_index])))
+
+            dates.append(
+                _round_to_seconds(
+                    start_date + unit.make_delta(float(vals[date_index]))
+                ),
+            )
+            # Due to https://github.com/equinor/ert/issues/6952
```

Comment:
Could you make this comment stand-alone? Otherwise I foresee a future where the below line will simply be deleted, as I assume that it is related to this?

Reply:
Fixed